### PR TITLE
Fix type and member naming

### LINKED ISSUE #909: Address S100: Methods and properties should be named in PascalCase
Address existing violations of [S100: Methods and properties should be named in PascalCase](https://rules.sonarsource.com/csharp/RSPEC-100) in the codebase and set severity to `Warning` in `Steeltoe.Debug.ruleset` and `Steeltoe.Release.ruleset`.

To find existing violations, enable the rule (see above) and rebuild `src/Steeltoe.All.sln` to make them appear in the Output window.

To address the violations, choose from the following on a case-by-case basis:
- Fix the violation by renaming the affected member
- Suppress the violation in code using `#pragma warning disable/restore`, in case it represents a well-known exception (for example: OSX)

### PR DESCRIPTION
**Pull request by bart-vmware**
## Description

Fixes #909.

## Quality checklist

- [x] Your code complies with our [Coding Style](https://github.com/SteeltoeOSS/.github/blob/main/contributing-docs/contributing-code-style.md).
- [ ] You've updated unit and/or integration tests for your change, where applicable.
- [ ] You've updated documentation for your change, where applicable.
      If your change affects other repositories, such as [Documentation](https://github.com/SteeltoeOSS/Documentation), [Samples](https://github.com/SteeltoeOSS/Samples) and/or [MainSite](https://github.com/SteeltoeOSS/MainSite), add linked PRs here.
- [x] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.
- [ ] You've added required license files and/or file headers (explaining where the code came from with proper attribution), where code is copied from StackOverflow, a blog, or OSS.

### CODE REVIEW

File: src/Common/src/Common.Expression/Spring/Standard/Tokenizer.cs [hunk 1]
```diff
@@ -688,4 +688,3 @@ private void RaiseParseException(int start, SpelMessage msg, params object[] ins
         throw new InternalParseException(new SpelParseException(_expressionString, start, msg, inserts));
     }
 }
-#pragma warning restore S125 // Sections of code should not be commented out
```
Comment (TimHess) [inline src/Common/src/Common.Expression/Spring/Standard/Tokenizer.cs:691]: What about the line (6) that starts the suppression?

File: src/Common/test/Common.Expression.Test/Spring/SpelReproTests.cs [hunk 2]
```diff
@@ -11,6 +11,11 @@
 using Steeltoe.Common.Util;
 using Xunit;
 
+// ReSharper disable InconsistentNaming
+#pragma warning disable S100 // Methods and properties should be named in PascalCase
+#pragma warning disable SA1307 // Accessible fields should begin with upper-case letter
+#pragma warning disable IDE1006 // Naming Styles
```
Comment (TimHess) [inline src/Common/test/Common.Expression.Test/Spring/SpelReproTests.cs:17]: Should these be turned back on later in the file?

File: src/Common/test/Common.Expression.Test/Spring/PerformanceTests.cs [hunk 3]
```diff
@@ -135,4 +135,3 @@ public void TestPerformanceOfMethodAccess()
         }
     }
 }
-#pragma warning restore xUnit1004 // Test methods should not be skipped
```
Comment (TimHess) [inline src/Common/test/Common.Expression.Test/Spring/PerformanceTests.cs:138]: What about the line that starts the suppression?

File: src/Common/test/Common.Expression.Test/Spring/SpelReproTests.cs (hunk 2 above)
Comment (bart-vmware) [reply src/Common/test/Common.Expression.Test/Spring/SpelReproTests.cs:17]: No, all preprocessor defines automatically go out of scope at the end of the file they were declared in.

File: src/Common/test/Common.Expression.Test/Spring/PerformanceTests.cs (hunk 3 above)
Comment (bart-vmware) [reply src/Common/test/Common.Expression.Test/Spring/PerformanceTests.cs:138]: The suppression is still needed, but there's just no point in restoring at the end of the file.

**Review by TimHess — APPROVED** [review]

File: src/Common/src/Common.Expression/Spring/Standard/Tokenizer.cs (hunk 1 above)
Comment (bart-vmware) [reply src/Common/src/Common.Expression/Spring/Standard/Tokenizer.cs:691]: I suspect I've answered this in the other responses. Did you mean something else?

Comment (TimHess) [reply src/Common/src/Common.Expression/Spring/Standard/Tokenizer.cs:691]: Just another instance of the same question. Feels like having something open and not close, but if there's no side affect then I'm not worried about it

Comment (bart-vmware) [reply src/Common/src/Common.Expression/Spring/Standard/Tokenizer.cs:691]: I get your point. With preprocessor directives its quite common to not 'close' them, in contrast to most other things. For example, consider `#define USE_TRACING` at the top of a file, which makes it easy to turn on/off during debug. If we were to 'close' that one, we'd have to undefine USE_TRACING at the end of the file, which nobody ever does.

**Pull request merged**